curriculum/challenges/english/25-front-end-development/workshop-greeting-card/6733352ce2a5642827b4f7f0.md

```diff
@@ -0,0 +1,125 @@
+---
+id: 6733352ce2a5642827b4f7f0
+title: Step 18
+challengeType: 0
+dashedName: step-18
+---
+
+# --description--
+
+Add a `.card-links` selector, and set the `margin-top` property to `20px`.
+
+Then, to align the two links, add a `display` property set to `flex`, and a `justify-content` set to `space-around`.
```

Comment:
Flexbox hasn't been taught yet. So we will need to add a brief explanation to what flexbox is and talk about what justify content and space-around does. Then mention they will learn more about flexbox in a future module.